src/models/degiroRecord.ts

```diff
@@ -8,5 +8,17 @@ export class DeGiroRecord {
     fx: string;
     currency: string;
     amount: string;
+    balance_currency: string; // not used, but improtant for hashing
+    balance: string; //// not used, but improtant for hashing
     orderId: string;
+
+    public getAmount(): Number { return parseFloat(this.amount.replace(",", ".")); }
+    public getAbsoluteAmount(): Number { return Math.abs(this.getAmount()); }
+
+    public getFX(): Number { return parseFloat(this.fx.replace(",", ".")); }
```

Comment:
In the time I have maintained the tool I haven’t found fx to be relevant until now. Why would it be?

Reply:
yeah, I think I added it for completeness. I can remove if you prefer.

Reply:
Yes please